src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,78 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
-public class HelloWorldTest {
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
+import core.basesyntax.db.Storage;
+import core.basesyntax.exaption.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User testUser;
+
+    @BeforeEach
+    void setUp() {
+        testUser = new User();
+        testUser.setLogin("QWERTY123");
+        testUser.setAge(21);
+        testUser.setPassword("start33Password_007");
+    }
+
+    @Test
+    void addTrueUser_Ok() {
+        assertEquals(registrationService.register(testUser), testUser);
+    }
+
+    @Test
+    void addUserIsRegister_exception() {
+        registrationService.register(testUser);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(testUser);
+        });
+    }
+
+    @Test
+    void addUserPasswordLessThen6_exception() {
+        testUser.setPassword("12345");
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(testUser);
+        });
+    }
```

Comment:
![image](https://github.com/mate-academy/jv-registration-validation/assets/63095184/f143be8c-ba1a-4f2c-877c-6e6cbfa9aac9)
Use naming for negative scenario tests